tests/ingest/ingest_with_text_embedding_processor.yaml

```diff
@@ -0,0 +1,108 @@
+$schema: ../json_schemas/test_story.schema.yaml
+
+skip: false
+description: |
+  This test story checks that we can create an ingest pipeline with a text 
+  embedding processor a uses it to create an index.
+epilogues:
+  - path: /_ingest/pipeline/books_pipeline
+    method: DELETE
+    status: [200, 404]
+  - path: /books
+    method: DELETE
+    status: [200, 404]
+  # - path: /_plugins/_ml/model_groups/{model_group_id}
+  #   method: DELETE
+  #   status: [200, 404]
+  #   parameters_from_environment:
+  #     model_group_id: test_model_group_id
+  - path: /_plugins/_ml/models/{model_id}
+    method: DELETE
+    status: [200, 404]
+    parameters_from_environment:
+      model_id: test_model_id
+chapters:
+  - synopsis: Configure cluster
+    path: /_cluster/settings
+    method: PUT
+    request_body:
+      payload:
+        persistent:
+          plugins:
+            ml_commons:
+              only_run_on_ml_node: false
+  # - synopsis: Create model group
+  #   path: /_plugins/_ml/model_groups/_register
+  #   method: POST
+  #   request_body:
+  #     payload:
+  #       name: "NLP_Group"
+  #       description: "Model group for NLP models"
+  #   remember:
```

Comment:
I like the way GitHub actions does this. How about an `output` that can be referenced?

```yaml
id: create_model_chapter
output:
   test_model_group_id: response_body.model_group_id
```

Then it could be referenced as `create_model_chapter.test_model_group_id`?